Jenkinsfile

```diff
@@ -1,11 +1,21 @@
 #!/bin/env groovy
 
-node {
-    stage ("Trigger") {
-        dir ("dlang/ci") {
-            git "https://github.com/dlang/ci.git"
-        }
+def cloneUpstream () {
+    checkout(scm: [
+        $class: 'GitSCM',
+        branches: scm.branches,
+        extensions: scm.extensions + [[$class: 'CleanBeforeCheckout']],
+        userRemoteConfigs: scm.userRemoteConfigs
+    ])
+}
 
-        load "dlang/ci/pipeline.groovy"
+def pipeline
+node {
+    dir('dlang/ci') {
+        cloneUpstream()
     }
+    pipeline = load 'dlang/ci/pipeline.groovy'
```

Comment:
This might circumvent the trusted Jenkinsfile source policy of Jenkins. Which is fine since the workers are encapsulated and there isn't much to protect on that server either.